src/main/java/com/ft/methodearticlemapper/transformation/EomFileProcessor.java

```diff
@@ -325,13 +333,18 @@ private AccessLevel getAccessLevel(final XPath xpath, Document attributes, UUID
     }
 
     private String determineType(final XPath xpath,
```

Comment:
This method should receive the content source as a parameter instead of the whole eomFile. Right now you also send the attributesDocument as a parameter which is part of the eomFile.